apps/cnspec/cmd/config/config.go

```diff
@@ -23,7 +23,7 @@ type CliConfig struct {
 
 	// Asset Category
 	Category               string `json:"category,omitempty" mapstructure:"category"`
-	AutoDetectCICDCategory bool   `json:"detect_cicd,omitempty" mapstructure:"detect_cicd"`
+	AutoDetectCICDCategory bool   `json:"detect-cicd,omitempty" mapstructure:"detect-cicd"`
```

Comment:
While I agree that we revert it, all config need to use `_`, we do not use `-` since it will not work with proto json parsing.